Sources/Nimble/Matchers/BeAKindOf.swift

```diff
@@ -2,13 +2,23 @@ import Foundation
 
 #if _runtime(_ObjC)
```

Comment:
Since this supports Swift types, it's probably useful to remove/adjust it's ObjC-runtime only build conditional?

(Also, don't forget to conditionally do this for tests).